dmcontent/__init__.py

```diff
@@ -2,4 +2,4 @@
 from .errors import ContentTemplateError, QuestionNotFoundError
 from .questions import ContentQuestion
 
-__version__ = '4.4.2'
+__version__ = '4.5.0'
```

Comment:
There's no new functionality added to content loader here, so I'd probably argue for a patch version bump.

Reply:
I'm going to stick with Minor because of the decision I came to on the utils PR (same/ similar changes are included here):

https://github.com/alphagov/digitalmarketplace-utils/pull/338/files
https://govuk.slack.com/archives/C04AXAXEM/p1506433136000301
>>> I'm going to make a call on this. It is my opinion that, technically, according to the semver spec, I've changed the interface for these packages, but not strictly broken backwards compatibility for _documented_ api functionality (both by removing imports and changing the Makefile commands). I'm going to release Flake8 changes with `Makefile` and import changes as minor version bumps, in line with the semver spec.



http://semver.org/